ztools/x509/x509.go

```diff
@@ -345,6 +345,19 @@ func getPublicKeyAlgorithmFromOID(oid asn1.ObjectIdentifier) PublicKeyAlgorithm
 	return UnknownPublicKeyAlgorithm
 }
 
+func getMaxCertValidationLevel(oids []asn1.ObjectIdentifier) CertValidationLevel {
+	maxOID := DV
+	for _, oid := range oids {
+		if _, ok := ExtendedValidationOIDs[oid.String()]; ok {
```

Comment:
Technically, yes. But since our sources (Chromium, Mozilla) identify/fingerprint certificates in different ways and CA/Browser (https://cabforum.org/object-registry/) doesn't provide matching certificate fingerprints at all, it's not clear if we would be able to do this accurately, and will probably end up with false negatives. 

On the other hand, if we don't confirm the issuer, we can end up with false positives, but from what I've seen, an issuer has no incentive to use a different CAs proprietary EV or OV OID. 

Side note - this might be a good cert linter task